### PR TITLE
Refactor/#41 standardize error messages

### PR DESCRIPTION
**Pull request by dopenguin**
## Summary

`console.warn` and `console.error` have been standardized to include the respective package at the beginning of the statement.

Also added an [ADR](https://github.com/Dataport/polar/wiki/Architecture-Decision-Records-%F0%9F%93%83#2024-01-10-console-statement-standardization) that enforces this change for all upcoming `console` statements.

I have specifically not touched `Error` messages, as they sometimes are used in toasts. Changing them would blow the proportions of these changes. If you feel standardizing those are important as well, we can tackle that separately.

## Instructions for local reproduction and review

Check whether the messages are in line with the ADR.

## Relevant tickets, issues, et cetera

#41

### CODE REVIEW

File: packages/plugins/AddressSearch/src/utils/searchMethods/getSearchMethod.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export const getMethodContainer = () => {
       return method
     }
     throw new Error(
-      `AddressSearch: The given type "${type}" does not define a valid searchMethod.`
+      `The given type "${type}" does not define a valid searchMethod.`
```
Comment (warm-coolguy) [inline packages/plugins/AddressSearch/src/utils/searchMethods/getSearchMethod.ts:29]:
```suggestion
      `@polar/plugin-address-search: The given type "${type}" does not define a valid searchMethod.`
```

Let's add/keep this here so configurators know where to check.

**Review by warm-coolguy — CHANGES_REQUESTED** [review]
🏓 @dopenguin

File: packages/plugins/AddressSearch/src/utils/searchMethods/getSearchMethod.ts (hunk 1 above)
Comment (dopenguin) [reply packages/plugins/AddressSearch/src/utils/searchMethods/getSearchMethod.ts:29]: > I have specifically not touched Error messages, as they sometimes are used in toasts. Changing them would blow the proportions of these changes. If you feel standardizing those are important as well, we can tackle that separately.

**Comment by dopenguin** [comment]
> 🏓 @dopenguin

🏓 @warm-coolguy

**Comment by warm-coolguy** [comment]
> > 🏓 @dopenguin
> 
> 🏓 @warm-coolguy

🏓 @dopenguin

**Comment by dopenguin** [comment]
> > > 🏓 @dopenguin
> > 
> > 
> > 🏓 @warm-coolguy
> 
> 🏓 @dopenguin

🏓 @warm-coolguy

**Comment by warm-coolguy** [comment]
> > > > 🏓 @dopenguin
> > > 
> > > 
> > > 🏓 @warm-coolguy
> > 
> > 
> > 🏓 @dopenguin
> 
> 🏓 @warm-coolguy

🚀 @dopenguin

**Pull request merged**